src/main/java/graphql/annotations/DefaultTypeFunction.java

```diff
@@ -300,66 +331,70 @@ public GraphQLType apply(Class<?> aClass, AnnotatedType annotatedType) {
                 return type;
             }
         }
-
-        @Override
-        public Collection<Class<?>> getAcceptedTypes() {
-            return Collections.singletonList(Object.class);
-        }
     }
 
     public DefaultTypeFunction() {
-        registry = new ConcurrentHashMap<>();
```

Comment:
why take this out.  Or at least make a concurrent list replacement.

People use this staticalyl today in some cases and hence thread visibility is important